SKILL.tsv

```diff
@@ -4819,7 +4819,7 @@ SKILL_20150717_004818	According to skill 5 seconds at MakuTaro bursts was 10% of
 SKILL_20150717_004819	The enemies that are shot by [Magic Arrow] will fall into [Silence] with a certain probabillity. The probability will increase by 10% per Attribute level.
 SKILL_20150717_004820	You can create the arrows for Fletcher. Crafting arrow is available while rest mode.
 SKILL_20150717_004821	{memo X}Defense decrease effect of [Deprotected Zone] increases by 1 per attribute level.
-SKILL_20150717_004822	$Increases the chance of stun by 2% per attribute level when using a basic attack on enemies with a [Blunt] weapon type.
+SKILL_20150717_004822	$Increases the chance of stun by 2% per attribute level when using a basic attack on enemies with a [Bludgeon] weapon type.
```

Comment:
This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].